src/webui/www/private/scripts/client.js

```diff
@@ -1738,6 +1738,19 @@ window.addEventListener("load", () => {
     window.qBittorrent.Cache.preferences.init();
     window.qBittorrent.Cache.qbtVersion.init();
 
+    // Setup color scheme switching
+    const updateColorScheme = () => {
+        const root = document.documentElement;
+        const colorScheme = LocalPreferences.get("color_scheme");
+        const validScheme = (colorScheme === "light") || (colorScheme === "dark");
+        const isDark = window.matchMedia("(prefers-color-scheme: dark)").matches;
+        root.classList.toggle("dark", ((!validScheme && isDark) || (colorScheme === "dark")));
+    };
+
+    const colorSchemeQuery = window.matchMedia("(prefers-color-scheme: dark)");
+    colorSchemeQuery.addEventListener("change", updateColorScheme);
+    updateColorScheme();
```

Comment:
Just suggesting to reuse `colorSchemeQuery`:
```suggestion
    // Setup color scheme switching
    const colorSchemeQuery = window.matchMedia("(prefers-color-scheme: dark)");
    const updateColorScheme = () => {
        const root = document.documentElement;
        const colorScheme = LocalPreferences.get("color_scheme");
        const validScheme = (colorScheme === "light") || (colorScheme === "dark");
        const isDark = colorSchemeQuery.matches;
        root.classList.toggle("dark", ((!validScheme && isDark) || (colorScheme === "dark")));
    };

    colorSchemeQuery.addEventListener("change", updateColorScheme);
    updateColorScheme();
```

Reply:
Done.